packages/coinstac-decentralized-algorithm-integration/test/index.js

```diff
@@ -8,10 +8,34 @@ let toRun = [
   'declarations/group-add.js',
   'declarations/plugin-group-step.js',
   'declarations/plugin-group-step-seeder.js',
-  'declarations/process-files.js',
```

Comment:
Have to comment this out. We’re solely relying on metadata CSVs for the time being, so declarations’ local data doesn't even enter the computations.